### PR TITLE
Add weights, move files: Tasks>AdminClust>NetPol.

### PR DESCRIPTION
**Pull request by steveperry-53**
Partially addresses #8398.

Preview: https://deploy-preview-8634--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy/

This is one of several PRs that I will submit to get the Tasks left nav in order.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/network-policy-provider/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Install a Netork Policy Provider
```
Comment (bradtopol) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/_index.md]: Small typo here: Netork should be Network

**Comment by bradtopol** [comment]
Just found a small typo.  It's very close though and looking good!

**Comment by steveperry-53** [comment]
@bradtopol, Fixed Nework typo.

**Comment by bradtopol** [comment]
Thanks @steveperry-53 Looks great!
/approve
/lgtm

**Comment by bradtopol** [comment]
@steveperry-53 Looks like you had some merge conflicts in two files.  They were very minor. I just resolved the conflicts using the the github web ui.  Feel free to double check but I think we are good now.
/approve
/lgtm

**Pull request merged**